cm2350/e200z7.py

```diff
@@ -137,8 +136,13 @@ def __init__(self, vw):
         # class the PPC_e200z7 is designed so that the core vivisect workspace
         # emulator can be removed in the future to improve performance (at the
         # cost of analysis/inspection/live debug capabilities).
+        vimp_emu.WorkspaceEmulator.__init__(self, self.vw, nostack=True, funconly=False)
```

Comment:
Where is `self.vw` coming from?  You removed `vw` from `__init__()`.  I assume this is requiring that the subclass populate that before initializing this?  That's at least a documentation requirement.

Reply:
As commented below I'll change this so it's no longer a required class item.